### ISSUE TITLE
Typo in util function errorcode_of_exception

### ISSUE THREAD

**Issue by SStembridge** [BingAds/BingAds-Python-SDK#126]
I'm not familiar with creating pull requests and it's a simple typo so I'd like to point it out here.
https://github.com/BingAds/BingAds-Python-SDK/blob/master/bingads/util.py
line 60
```
    def errorcode_of_exception(ex):
        if isinstance(ex, WebFault):
            if hasattr(ex.fault, 'detail') \
                    and hasattr(ex.fault.detail, 'AdApiFaultDetail') \
                    and hasattr(ex.fault.detail.AdApiFaultDetail, 'Errors') \
                    and hasattr(ex.fault.detail.AdApiFaultDetail.Errors, 'AdApiError'):
                ad_api_errors = ex.fault.detail.AdApiFaultDetail.Errors.AdApiError
                if type(ad_api_errors) == list:
==>                return ad_api_erros[0].Code
                else:
                    return ad_api_errors.Code
        return -1
```
ad_api_erros[0] should be ad_api_errors[0]

**Comment by qitia** [comment]
thanks @SStembridge. Will update it soon.

**Comment by eric-urban** [comment]
@SStembridge this is resolved via [release 12.13.3.1](https://github.com/BingAds/BingAds-Python-SDK/commit/4a0530aeca6653af667276eda51f48e08e2a6b21). Thanks for the catch!

@qitia thanks for the quick update!
